zeppelin-plugins/launcher/docker/src/main/java/org/apache/zeppelin/interpreter/launcher/DockerInterpreterProcess.java

```diff
@@ -340,16 +353,28 @@ public Void call(RemoteInterpreterService.Client client) throws Exception {
 
       // Remove container
       docker.removeContainer(containerName);
-    } catch (DockerException e) {
-      e.printStackTrace();
-    } catch (InterruptedException e) {
-      e.printStackTrace();
+    } catch (DockerException | InterruptedException e) {
+      LOGGER.error(e.getMessage(), e);
     }
 
     // Close the docker client
     docker.close();
   }
 
+  private void removeExistContainer(String containerName) {
+    try {
+      docker.killContainer(containerName);
+    } catch (DockerException | InterruptedException e) {
+      LOGGER.error(e.getMessage(), e);
```

Comment:
should not `error` if docker doesn't exist?

Reply:
done.
Increase the judgment of whether there is a container with the same name.